pennylane/qnn/torch.py

```diff
@@ -448,20 +450,17 @@ def construct(self, args, kwargs):
 
     def __getattr__(self, item):
         """If the given attribute does not exist in the class, look for it in the wrapped QNode."""
-        if self._initialized:
-            return getattr(self.qnode, item)
+        with contextlib.suppress(AttributeError):
+            if self._initialized:
```

Comment:
I'd swap these because why not
```suggestion
        if self._initialized:
            with contextlib.suppress(AttributeError):
```